husky_description/urdf/generate_dual_arm_husky_urdf.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+rosrun xacro xacro.py dual_arm_husky.urdf.xacro laser_enabled:=false ur5_enabled:=false kinect_enabled:=false dual_ur5_enabled:=true robotiq_grippers_enabled:=true > dual_arm_husky.urdf
```

Comment:
I like this. Shows users how to build it. Not how we usually do things - but I like it.